app/src/main/java/me/guillaumin/android/osmtracker/activity/TrackDetail.java

```diff
@@ -238,6 +239,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
 			data.put(ITEM_VALUE, DateFormat.getDateTimeInstance().format(new Date(System.currentTimeMillis())));
 			adapter.notifyDataSetChanged();
 			break;
+		case R.id.trackdetail_menu_share:
+			new ExportAndShareTask(this, trackId).execute();
+			// Pick last list item (Exported date) and update it
+			SimpleAdapter adapter1 = ((SimpleAdapter) lv.getAdapter());
```

Comment:
Would you mind avoiding names like `adapter1`, prefer just `adapter` for instance (see what's done for the other menus)?